### PR TITLE
Blog app - Views and Navigations

### PR DESCRIPTION
**Pull request by Enoisong**
## Requirements:
-  Implement the design from the sneak peek wireframes.🐚✏
-  Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone) 🏃‍♂️🏃‍♂️
      -  **Note**: Analyze the wireframes deeply and try to identify all the necessary elements and how a user would navigate from one page to another✔📄
-  Use methods already created in the [Blog App - processing data in models]🐚🤸‍♂️
(https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
      -  **Hint**: In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).✍🤸‍♂️

## Achievements

- Add views for each controller as per the wireframe ✍🤸‍♂️
- Add Navigations✔📄
- Update rspec✏
- Add style🐚
- Fix Linters.⛏

### CODE REVIEW

File: app/views/shared/_user-container.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class='user-container'>
+  <img src="<%= user.photo%>" class='user-img' />
+  <div class='user-div'>
+    <h2><%= link_to user.name, user_url(user) %></h2>
+    <small class='posts_counter'>Number of posts: <%= user.posts_counter %></small>
+  </div>
```
Comment (VICTOR-CHUKWU) [inline app/views/shared/_user-container.html.erb]: - [ ] kindly keep to design wireframe, even after adding user photo in the seed file, I still could not see the picture added, kindly fix
your design :point_down: 
Screenshot from 2023-08-31 20-24-55
wireframe :point_down: 
Screenshot from 2023-07-20 22-19-45

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,16 @@
...
+      <p><%= post.text %></p>
+      <small>Comments:<%= post.comments_counter %>, Likes:<%= post.likes_counter %></small>
+    </div>
+<% end %>
+<div class='button-wrapper'>
+  <button type="button" class='see-posts-btn'>
+    <%= link_to "See all posts", user_posts_url(@user)%>
+  </button>
+</div>
+</div>
```
Comment (VICTOR-CHUKWU) [inline app/views/users/show.html.erb]: - [ ] also kindly keep to the given wireframe 
your design :point_down: 
Screenshot from 2023-08-31 20-31-01
design guide :point_down: 
Screenshot from 2023-07-20 22-23-13

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,17 @@
-<h1>List of all posts from the given user id</h1>
\ No newline at end of file
+<div class='template'>
+  <%= render partial: 'shared/user-container', locals: { user: @user } %>
+  <% @posts.each do |post| %>
+    <div class='post-container'>
+      <div>
+        <h3><%= link_to post.title, user_post_url(@user, post) %></h3>
```
Comment (VICTOR-CHUKWU) [inline app/views/posts/index.html.erb:5]: - [ ] also the profile picture is not showing, kindly fix it as to fit the wireframe

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Enoisong  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed
- views not fitting perfectly yet


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Enoisong** [comment]
## Required Changes Implementation👨‍💻

## Requirements

-  The profile picture is not showing, kindly fix it as to fit the wireframe.

-  Kindly keep to design wireframe, even after adding user photo in the seed file, I still could not see the picture added, kindly fix.

- kindly keep to the given wireframe.
 
## Achievements
Add profile picture to the app with alt image (profile)✔

All Implementations of the design follows the sneak peak wireframes✔

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Enoisong 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- Now the User picture is showing :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**